eliot/tests/strategies.py

```diff
@@ -40,11 +44,11 @@
 
 
 # Text generation is slow, and most of the things are short labels.
-labels = text(average_size=5)
+labels = text(average_size=3, min_size=1, alphabet="CGAT")
```

Comment:
I take it smaller alphabets make it quicker to generate? perhaps expand comment.